src/components/some-new-component/some-new-component.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="some-new-component">
+    <p>Some new component for step: {{ viewerCurrentStep.title }}</p>
+  </div>
+</template>
+
+<script>
+  import { mapGetters } from 'vuex';
+
+  export default {
+    data() {
+      return {
+        foo: 'bar',
+      };
+    },
+    computed: {
+      ...mapGetters('app', [ 'viewerCurrentStep' ]),
+    },
+  };
+</script>
+
+<style src="./some-new-component.css" lang="css"></style>
```

Comment:
Does this serve as an example component or is it a leftover?

Reply:
As an example for us. It will probably already be removed when we start working on the next tickets.